monfrontend/app.py

```diff
@@ -130,6 +151,43 @@ def fetch_data(intervals):
         }
     return measurements_data
 
+@app.callback(
+    Output("machine-status", "children"), [Input("in-memory-storage", "data"),],
+)
+def live_status(data):
+    """
+    Generates live machine parameters as children of div 'machine-parameters'
+    """
+    measurements = list(influx.get_measurements())
+    # for now use all available measurements, instead data.keys?
+
+    children = []
+    for msmt in measurements:
+        children.append(
+            html.H6(f"MachineStatus:{1}", className="motor_status"),
+        )
+    return children
+
+@app.callback(
+    Output("machine-parameters", "children"), [Input("in-memory-storage", "data"),],
+)
+def live_machine(data):
+    """
+    Generates live machine parameters as children of div 'machine-parameters'
+    """
+    measurements = list(influx.get_measurements())
+    # for now use all available measurements, instead data.keys?
+
+    children = []
+    for msmt in measurements:
```

Comment:
Here as well, 5 strings per measurement that is in the database

Reply:
No that's not intended. Wrong Syntax/placeholder. Needs to be mapped to actual status values transmitted.